### DISCUSSION TITLE
Installed WinGet packages disappear again

### DISCUSSION THREAD

**Discussion by Frank-Radmayr** [Romanitho/Winget-Install#87] [Q&A]
Hello together
I want to distribute WinGet SW packages via SCCM to the clients in our company.
In the meantime I have packaged 10 WinGet packages and distributed them to several test clients via SCCM.
The install Powershell command looks +- the same for all SW packages

$appID = '9NBLGGH5R558'
winget install --silent --accept-source-agreements --accept-package-agreements --source msstore --id $appID

Now the amazing, after a few days, all WinGet packages are deleted or uninstalled from the clients as if by magic.
I don't see any entries in any log, be it the WinGet specific App Log, neither nor in the eventvwr.

Does anyone have any experience or ideas as to why these WinGet apps are disappearing from the client?

Thanks
Kind regards
Frank

**Comment by MichaelReiserSAP** [comment]
Looks like this is related to a outdated winget client. I do look into it right now.

**Reply by Frank-Radmayr** [reply]
The Winget version we have in use is 1.6.2631

**Reply by MichaelReiserSAP** [reply]
Meanwhile I'm sure that it's a problem on Microsoft side. It looks like winget can't connect to the repository. Others do report this issue too.

**Reply by Frank-Radmayr** [reply]
Does a solution already exist for this problem?
Is it also meant that if a client is not connected to the internet for a "long" time, the WinGet packages would also be uninstalled from the client, is that so?

**Reply by MichaelReiserSAP** [reply]
Not that I know. Assuming MS is working on the issue. It looks like this did happen before. The second question I can't answer.

**Reply by MichaelReiserSAP** [reply]
https://github.com/microsoft/winget-cli/issues/3652

**Reply by Romanitho** [reply]
not sure this is the correct place for your issue :/

**Reply by Frank-Radmayr** [reply]
what place would you suggest then?

**Reply by Romanitho** [reply]
into the Microsoft Repository https://github.com/microsoft/winget-cli/issues
